tests/test_ui_extra.py

```
@@ -87,3 +87,60 @@ def capture_on(evt, handler):
+    with patch('structui.ui.ui.notify') as mock_notify, patch('structui.ui.ui.button'):
+        with patch('structui.ui.LocalFilePicker') as mock_picker:
+            async def pick_mock(*args, **kwargs):
+                return ["/mock/dir/path"]
+            mock_picker.side_effect = pick_mock
+
+            mock_app_state.data_dir = "/mock/dir/path"
+            mock_app_state.load_files.side_effect = Exception("Load error")
+
+            try:
+                mock_app_state.load_files()
+            except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)